llvm/lib/Target/X86/X86InstrMisc.td

```diff
@@ -1534,7 +1534,7 @@ let SchedRW = [WriteStore], Defs = [EFLAGS] in {
   def ENQCMD64 : I<0xF8, MRMSrcMem, (outs), (ins GR64:$dst, i512mem_GR64:$src),
                  "enqcmd\t{$src, $dst|$dst, $src}",
                  [(set EFLAGS, (X86enqcmd GR64:$dst, addr:$src))]>,
-                 T8, XD, AdSize64, Requires<[HasENQCMD, In64BitMode]>;
+                 T8, XD, AdSize64, Requires<[HasENQCMD, NoEGPR, In64BitMode]>;
 
```

Comment:
Rewrite it with `ITy` first

Reply:
This may hard to be rewrited because they don't use w/l/q suffix, or I need to add 1 more para to ITy to indicate if this need size suffix.

Reply:
Ok, I didn't notice that.